oak/server/oak_node.h

```diff
@@ -44,13 +44,6 @@ class OakNode final : public Application::Service {
   // running the instance completes.
   virtual ~OakNode();
 
-  // Performs an Oak Module gRPC invocation. Takes ownership of the passed in request data.
-  void ProcessModuleInvocation(grpc::GenericServerContext* context,
-                               std::unique_ptr<Message> request_data);
```

Comment:
Can we now drop the `#include` (and BUILD dep) of `grpc_encap.h` ?

Reply:
Yes.